test/memo_test.rb

```diff
@@ -0,0 +1,166 @@
+# frozen_string_literal: true
+
+require 'minitest/autorun'
+require 'rack/test'
+require_relative 'constants'
+require_relative '../memo'
+
+class MemoTest < Minitest::Test
+  include Rack::Test::Methods
+
+  def app
+    Sinatra::Application
+  end
+
+  def setup
+    File.exist?('./db/memos_test.json') && File.delete('./db/memos_test.json')
+    memos = {
+      "a90cc4d4-8a66-4fc6-8609-a02f7fe0cf96": {
+        "public_id": 'a90cc4d4-8a66-4fc6-8609-a02f7fe0cf96',
+        "title": '買い物リスト',
+        "content": 'トマトジュース\nティッシュペーパー'
+      },
+      "b716320e-99d4-4050-bbf7-3c9b26a64665": {
+        "public_id": 'b716320e-99d4-4050-bbf7-3c9b26a64665',
+        "title": 'スクラム本',
+        "content": 'SCRUM BOOT CAMP\nアジャイルプラクティスガイドブック'
+      }
+    }
+    write_memos_table(memos)
+  end
+
+  def test_index
+    get '/memos'
+    assert last_response.status, 200
+    assert last_response.body.include?('買い物リスト')
```

Comment:
```suggestion
    assert_includes last_response.body, '買い物リスト'
```

みたいに書けるかも？（テストに失敗したときにわかりやすく表示されそう）

Reply:
62181d9 にて対応しました！